database/seeders/ThreeDotsSeeder.php

```
@@ -96,9 +83,22 @@ private function talks(Tenant $tenant, EventModel $event): void
                     'field_type' => $item['field_type'],
                     'title' => $item['title'],
                     'description' => $item['description'],
-                    'starts_at' => Date::parse($item['time']),
-                    'ends_at' => Date::parse($item['time']),
                 ]);
+
+            $talks = Talk::all()->toArray();
+
+            $segments = EventSegment::all()->toArray();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$segments'. (undefined)

(UnusedLocalVariable)